build.assets/buildbox/Dockerfile

```diff
@@ -100,6 +100,31 @@ RUN curl --proto =https --tlsv1.2 -fsSL https://sh.rustup.rs | \
 		arm-unknown-linux-gnueabihf \
 		wasm32-unknown-unknown
 
+# ----------------------------------------------------------------------------
+# Clang 12.0.0 for FIPS builds of boring-rs
+
+FROM base AS clang
+
+# libtinfo5 required to run clang to test it works.
+# xz-utils for decompressing the clang tarball.
+RUN apt-get update \
+    && apt-get install -y libtinfo5 xz-utils \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN install -d -m 0775 -o buildbox -g buildbox /opt/clang
+USER buildbox
+
+# TODO(camscale): Verify signature of download.
```

Comment:
Does this need to be addressed now?

Reply:
I was planning to address it for this, rust and go, I can't see yet how to verify Rust and Go installs. This can come after I think.